### DISCUSSION TITLE
PNG TFT for ST7789V RP2040

### DISCUSSION THREAD

**Discussion by shamirtahir50** [Bodmer/TFT_eSPI#2088] [Raspberry Pi Pico]
I am working with 16-bit TFT (Driver ST7789V) with RP2040. 

I have generate a bitmap of png from,
http://www.rinkydinkelectronics.com/t_imageconverter565.php

My PNG,
![af2cc020-a6bf-460c-af6e-e30b045f080a](https://user-images.githubusercontent.com/66121435/195981628-f707c97b-1613-431a-81c6-a0e5ab61b562.jpg)

But my tft is displaying it like this,
![IMG_20221015_152041](https://user-images.githubusercontent.com/66121435/195981689-94440d68-b31d-436d-acae-046c572ff670.jpg)

But same bitmap is working fine for TFT lcd for Arduino.
Whats the problem here? @Bodmer

**Comment by Bodmer** [comment]
Run the Colour_Test example and check if they reported colours are correct. If not you need to define the colour order in the setup file. See comments in User_Setup.h file for the required #define.

**Reply by shamirtahir50** [reply]
> Run the Colour_Test example and check if they reported colours are correct. If not you need to define the colour order in the setup file. See comments in User_Setup.h file for the required #define.

Well that solution worked fine. So Thank you.
